app/routes/events/EventAbacardRoute.js

```diff
@@ -16,10 +16,10 @@ import { getRegistrationGroups } from 'app/reducers/events';
 
 const searchTypes = ['users.user'];
 
-const loadData = (props, dispatch) => {
+const loadData = async (props, dispatch) => {
   const query = props.location.query.q;
   if (query) {
-    dispatch(autocomplete(query, searchTypes));
+    await dispatch(autocomplete(query, searchTypes));
```

Comment:
Why is this async? Wouldn't returning the promise from `dispatch` be sufficient?

Reply:
That works too, but then a `return Promise.resolve()` is needed in the end (if `!!query === false`)

Reply:
Ah, right. I guess async/await is cleaner then